.github/workflows/ci.yml

```diff
@@ -38,7 +38,8 @@ jobs:
             { "name": "rhel-9", "arch": "amd64", "tag": "latest" },
             { "name": "sles-15", "arch": "amd64", "tag": "latest" },
             { "name": "ubuntu-20.04", "arch": "amd64", "tag": "latest" },
-            { "name": "ubuntu-22.04", "arch": "amd64", "tag": "latest" }
+            { "name": "ubuntu-22.04", "arch": "amd64", "tag": "latest" },
+            { "name": "ubuntu-24.04", "arch": "amd64", "tag": "latest" }
```

Comment:
>{ "name": "ubuntu-24.04", "arch": "amd64", "tag": "latest" } [](http://example.com/codeflow?start=12&length=60)

Ubuntu 24.04 is not an official target for us yet (I get that is for Compliance aka CIS and STIG?)